tests/unittests/Foundation/ReferenceFoundation/TestNSCharacterSet.mm

```diff
@@ -98,11 +98,10 @@
 TEST(NSCharacterSet, Mutables) {
     auto attachmentCharacterUnichar = static_cast<unichar>(0xFFFC);
```

Comment:
You can only assume this will be little endian on little endian platforms. If you want to be 100% sure, please encode it as separate LE bytes, `const unsigned char attachmentCharacterUnicharBytes[] = {0xFC, 0xFF};`